xml/Microsoft.VisualBasic.Vsa/VsaEngine.xml

```diff
@@ -301,7 +302,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets a Boolean value that signifies whether the script engine produces debug information when the Compile method is called.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the script engine produces debug information; otherwise, <see langword="false" />.</value>
```